src/poliastro/core/angles.py

```diff
@@ -149,7 +149,7 @@ def nu_to_E(nu, ecc):
     The implementation uses the half-angle formula from [3]_:
 
     .. math::
-        E = 2 \atan \left( \sqrt{\frac{1 - e}{1 + e}} \tan{\frac{\nu}{2}}
+        E = 2 \operatorname{arctan} \left ( \sqrt{\frac{1 - e}{1 + e}} \tan{\frac{\nu}{2}} \right)
```

Comment:
Don't we have `\arctan` in LaTeX? Can you give it a try locally?

Reply:
Sure ... checking & confirming the same

Reply:
Yes does work ... changed `\operatorname{arctan}` to `\arctan`

Reply:
For reference ... Inverse Hyperbolic Tangent Function does need `\operatorname{arctanh}` for parsing into Latex :)